features/react-component-check.feature

```diff
@@ -84,3 +84,20 @@ Feature: React Component Check
     Then I should see that the "[data-search-type='people']" element exists
     Then I should see that the "[data-search-type='people_departments']" element exists
     Then I should see that the "[data-search-type='departments']" element exists
+
+  @api @javascript @anchor_menu__test
+  Scenario: Verify Anchor menu component (components-core)
+    Given I am an anonymous user
+    When I am at '/anchor-menu'
+    Then I wait for 2 seconds
+    #Check "On this page" appear
+    Then I should see that the ".uds-anchor-menu-wrapper" element exists
+    And I should see the heading "On This Page:"
+    #Check “current” item highlighted
+    Then I click "Block B"
+    And I wait for 1 seconds
+    And I scroll "#webspark-anchor-link--152" into view
+    And I wait for 1 seconds
```

Comment:
These steps are only necessary because the anchor link does not work correctly. Previously, the sticky menu took the place of the header menu. But, now both display as sticky, and the anchor link does not take this into account.  There is a ticket that was created for this bug: https://asudev.jira.com/browse/WS2-1961. So, until that gets fixed, I guess we will need these lines. But, eventually they should not be needed.

Reply:
This test is now removed in PR: https://github.com/ASUWebPlatforms/webspark-ci/pull/415